### PR TITLE
Navigation Buttons for small displays

### PR DESCRIPTION
**Pull request by MirekSobota**
**In this update:**

~- `flex-grow `added to `max-width: 767px.`~
- `justify-content: space-between` added to `max-width: 767px.`

 Please take a look and tell me what you think!

### CODE REVIEW

File: src/common/Navigation/styled.js
Comment (DorotaZielinska) [inline src/common/Navigation/styled.js:1]: What do you think about to change flex-grow to justify-content: space-between.

Comment (MirekSobota) [reply src/common/Navigation/styled.js:1]: I will try it! Thanks for good tip!

Comment (DorotaZielinska) [reply src/common/Navigation/styled.js:1]: Now looks better:D

**Review by DorotaZielinska — APPROVED** [review]

**Review by PawelOwiesek — APPROVED** [review]
It looks very good.

**Pull request merged**